### PR TITLE
Sap system details improvements

### PR DESCRIPTION
**Pull request by nelsonkopliku**
As a followup for https://github.com/trento-project/web/pull/240 and this comment https://github.com/trento-project/web/pull/240#pullrequestreview-921417245, here's some improvements for the SAP systems and HANA Databases details.

image

image

Some chores have been addressed like:
- importing modules with `@components/`, `@lib/`, `@sate/`
- Allow the Pill component to be defined the rounding mode
- update `package-lock.json`

### CODE REVIEW

File: assets/js/state/sagas/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
   updateClusterHealth,
   startClustersLoading,
   stopClustersLoading,
-} from '../clusters';
+} from '@state/clusters';
```
Comment (fabriziosestito) [inline assets/js/state/sagas/index.js:23]: nice

File: assets/js/state/selectors/index.js [hunk 2]
```diff
@@ -68,6 +69,20 @@ const enrichInstances = (instances, sapSystemId, state) =>
       };
     });
 
+export const findInstancesOnHost = (hostId) => (state) => {
```
Comment (fabriziosestito) [inline assets/js/state/selectors/index.js:72]: why find and not `get` ?

Comment (nelsonkopliku) [reply assets/js/state/selectors/index.js:72]: my bad, I broke consistency :sweat_smile:  
Gonna normalize those in `getSomething`.

Comment (nelsonkopliku) [reply assets/js/state/selectors/index.js]: Changed here https://github.com/trento-project/web/pull/244/commits/6dba40e69436bf0ac5af3ade780889ffd7788912

**Review by fabriziosestito — APPROVED** [review]
:shipit:  🚀

File: assets/js/components/DatabaseDetails/DatabaseDetails.jsx [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
 import { useParams } from 'react-router-dom';
 import { useSelector } from 'react-redux';
-import ListView from '@components/ListView';
-import Table from '@components/Table';
-import {
-  databaseHostsTableConfiguration,
-  databaseInstancesTableConfiguration,
-} from './tableConfigs';
 import { getDatabaseDetail } from '@state/selectors';
+import GenericSystemDetails from '@components/SapSystemDetails/GenericSystemDetail';
```
Comment (dottorblaster) [inline assets/js/components/DatabaseDetails/DatabaseDetails.jsx:5]: We should play a bit with `import` and `export` using `SapSystemDetails`'s `index.js` to allow this:

```js
import { GenericSystemDetails } from '@components/SapSystemDetails'
```

Comment (nelsonkopliku) [reply assets/js/components/DatabaseDetails/DatabaseDetails.jsx:5]: I am able to achieve that by changing `./assets/js/components/SapSystemDetails/index.js` like this

```js
import { GenericSystemDetails as SystemDetails } from './GenericSystemDetails';
import SapSystemDetails from './SapSystemDetails';

export const GenericSystemDetails = SystemDetails;

export default SapSystemDetails;
```

I am a bit unsure about the aliasing `SystemDetails` which is needed to not make collide what we're importing with what  we're exporting (`GenericSystemDetails`).
Probably there's a better way of doing it, which I don't know by now :sweat_smile: 

But it works and allows us to do what you're looking for

```js
import { GenericSystemDetails } from '@components/SapSystemDetails'
```

Thoughts?

Comment (nelsonkopliku) [reply assets/js/components/DatabaseDetails/DatabaseDetails.jsx:5]: I think I found a cleaner way to do it, without unnecessary aliasing

```js
import SapSystemDetails from './SapSystemDetails';

export { GenericSystemDetails } from './GenericSystemDetails';
export default SapSystemDetails;
```

feels better :smile:

Comment (nelsonkopliku) [reply assets/js/components/DatabaseDetails/DatabaseDetails.jsx]: gone this way https://github.com/trento-project/web/pull/244/commits/e734303f8d56d7b0c18c2ef25d7194c7918b0ddb

**Review by dottorblaster — APPROVED** [review]
Marvellous!

**Pull request merged**